timesketch/api/v1/resources.py

```diff
@@ -1046,11 +1046,11 @@ def post(self):
                 db_session.commit()
 
             # Start Celery pipeline for indexing and analysis.
-            if current_app.config.get(u'ENABLE_INDEX_ANALYZERS'):
-                from timesketch.lib import tasks
-                pipeline = tasks.build_index_pipeline(
-                    file_path, timeline_name, index_name, file_extension)
-                pipeline.apply_async(task_id=index_name)
+            from timesketch.lib import tasks
```

Comment:
shouldn't the import be at top?

Reply:
or perhaps have a comment like you've got below to explain that this is to prevent circular imports

Reply:
Yep, added comment. this is because how Celery works with the factory pattern I'm using.